src/main/java/blockchain/NotaryService.java

```diff
@@ -45,6 +45,7 @@ public BigInteger verify(byte[] checksum, String address) throws Exception {
         return this.contract.verify(address, checksum).send();
     }
 
+    //TODO does this has to be checksum, address not address, checksum like method above
     public BigInteger verify(File file, String address) throws Exception {
```

Comment:
address of the user to verify, is located in the wallet file and in the userinfo